mongorover/MongoCollection.lua

```diff
@@ -270,15 +270,15 @@ local MongoCollection = {__mode="k"}
 	-- Example usage at @{indexing.lua}.
 	-- @tparam table of fields and their corresponding index types
 	-- @tparam table of options to be used when creating index
-	-- @return string index name on success
+	-- @return string index name
 	function MongoCollection:createIndex(keys,opt)
 	    return self.collection_t:collection_create_index(luaBSONObjects, keys, opt)
 	end
 
 	---
 	-- Delete an index for a specified collection.
 	-- Example usage at @{indexing.lua}.
-	-- @tparam string an index name to be deleted
+	-- @tparam string an index name or original table with indexed fields and their corresponding types
```

Comment:
It looks like pymongo doesn't return anything. Source code [here](https://github.com/mongodb/mongo-python-driver/blob/c0767a551ac38fb8b630214bbf20368951e7609f/pymongo/collection.py#L1586).

I also ran it in python myself to verify:

```
>>> client = pymongo.MongoClient()
>>> client.foo.bar.create_index('a')
u'a_1'
>>> ret = client.foo.bar.drop_index('a_1')
>>> ret
>>> print(ret)
None
>>>
```

Don't worry, I don't expect you to do know all of this or to install pymongo to test things out. It's most likely my responsibility to make sure we're being consistent with pymongo in these cases.